themes/Clearlooks/lxqt-panel.qss

```diff
@@ -290,6 +298,151 @@ QMenu::indicator:non-exclusive:checked:selected {
     background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #a0c0e0, stop:1 #728eb9);
 }
 
+/*
+ * Fancy Menu
+*/
+#FancyMenu {
+    margin-left: 1px;
+    margin-right: 0px;
+    margin-top: 1px;
+    margin-bottom: 1px;
+    qproperty-icon: url(mainmenu.svg);
+    background-color: transparent;
+    color: #1a1a1a;
+}
+
+#FancyMenu:pressed {
+    margin-left: 0px;
+    margin-right: 0px;
+    margin-top: 0px;
+    margin-bottom: 0px;
+    background-color: transparent;
+    color: #1a1a1a;
+}
+
+#FancyMenu:hover {
+    margin-left: 1px;
+    margin-right: 0px;
+    margin-top: 1px;
+    margin-bottom: 1px;
+    background-color: transparent;
+    color: #1a1a1a;
+}
+
+LXQtFancyMenuWindow {
+    background: #ffffff;
+}
+
+#FancyMenu QLineEdit {
+    color: #000000;
+    background: #ffffff;
+    border: 1px solid #909090;
+    border-radius: 3px;
+    padding: 2px;
+    selection-background-color: #80a0d0;
+    selection-color: white;
+}
+
+#FancyMenu #AppView {
+    background: #f0f0f0;
+}
+#FancyMenu #CategoryView {
+    border: 1px solid grey;
+}
+#FancyMenu QListView::item {
+    border: 1px solid transparent;
+    padding: 0px;
+    margin: 0px;
+}
```

Comment:
`color: #1a1a1a;` is missing here. Without it, there's light text on light background with dark Qt styles.

Reply:
Clearlooks is weird with dark palettes anyway (volume and brightness popup follow palette and are dark), so I jumped this, but will add it. IMO we should load the default theme when clearlooks, frost, light or kde is chosen. https://github.com/lxqt/lxqt-config/issues/974

Reply:
IMO, the default palettes should to be avoided as far as possible -- unless in themes like the system theme, and if I correctly remember, in Frost, which is supposed to inverse them.